Sketch.java

```diff
@@ -2,35 +2,95 @@
 
 public class Sketch extends PApplet {
 	
-	
   /**
-   * Called once at the beginning of execution, put your size all in this method
-   */
-  public void settings() {
-	// put your size call here
-    size(400, 400);
+  * A program Sketch.java that sketches out a cat based on the parameters in the method.
+  * @author: D. Gu 
+  */
+  
+  public void settings(){
+    // Size of sketchpad
+    size(900, 900);
   }
 
-  /** 
-   * Called once at the beginning of execution.  Add initial set up
-   * values here i.e background, stroke, fill etc.
-   */
-  public void setup() {
-    background(210, 255, 173);
+  public void draw(){
+    drawCat(500, 500);
+    drawCatTwo(200, 200, 223, 100, 100, 100);
```

Comment:
while `drawCatTwo` serves a purpose in drawing a cat, it does not serve as an appropriate demonstration of methods that return a value since there is no functional use of the return value here (you do nothing with the `int` that is returned).